packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -204,6 +207,8 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
      */
     private _tombstoned = false;
     public get tombstoned() { return this._tombstoned; }
+    /** If true, throw an error when a tombstone data store is used. */
+    private readonly shouldThrowOnTombstoneUsage: boolean;
```

Comment:
I wonder if `throwOnTombstoneUsage` is clear enough. Another mention below

Reply:
Renamed.